Dockerfile

```diff
@@ -0,0 +1,11 @@
+from python:3
```

Comment:
actually it would be cool to have a wrapper script that takes the desired Python version as argument. or rather embed the `Dockerfile` content in a script.